From 74d813db2c7c88705b434371c032997667d857a5 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 06 Aug 2018 14:26:20 +0300
Subject: [PATCH] Add FW loader support fixup

Realy declare our self as HOST otherwise
we won't be built by defconfig (Makefile supports host build only)
---

diff --git a/configs/dali_defconfig b/configs/dali_defconfig
index fa413a8..646fab5 100644
--- a/configs/dali_defconfig
+++ b/configs/dali_defconfig
@@ -64,4 +64,4 @@
 BR2_PACKAGE_DSV=y
 BR2_PACKAGE_THERMAL=y
 BR2_PACKAGE_PREBOOT=y
-BR2_PACKAGE_FW_LOADER=y
+BR2_PACKAGE_HOST_FW_LOADER=y
diff --git a/package/fw_loader/Config.in.host b/package/fw_loader/Config.in.host
index 03964de..78f2da9 100644
--- a/package/fw_loader/Config.in.host
+++ b/package/fw_loader/Config.in.host
@@ -1,4 +1,4 @@
-config BR2_PACKAGE_FW_LOADER
+config BR2_PACKAGE_HOST_FW_LOADER
 	bool "FW Loader"
 	select BR2_PACKAGE_SPECS
 	select BR2_PACKAGE_SPDLOG
